html-css/week1/bayram/portfolio/index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Welcome to My Personal Portfolio!</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <nav>
+      <ul>
+        <li><a href="#about">About</a></li>
+        <li><a href="#skills">Skills</a></li>
+        <li><a href="#portfolio">Portfolio</a></li>
+        <li><a href="#contact">Contact</a></li>
+      </ul>
+    </nav>
+
+    <header>
+      <h1>Bayram Erdem</h1>
+      <p>Junior Developer</p>
+    </header>
+
+    <section id="about">
+      <h2>About Me</h2>
+      <p>
+        👋 Hi, I’m Bayram, an expat in Copenhagen. Coding isn’t just my
+        career—it’s my passion! I’m all about exploring new languages and adding
+        them to my toolkit. Right now, I’m diving into CSS and JavaScript, and
+        I’m hungry for more.
+      </p>
+      <p>
+        As a HYF participant, I’m excited to unlock the wonders of Frontend,
+        Backend Development Libraries, and even Python. 💡
+      </p>
+      <p>
+        <strong>I’m looking to:</strong> Collaborate on frontend projects. Find
+        an internship in tech that challenges me and lets me make an impact. Ask
+        me about how synergy works in a team! I love bringing people together
+        and organizing highly efficient teams—it’s complex, but that’s what
+        makes it so exciting.
+      </p>
+      <p>📫 Reach out on LinkedIn or email me at bayram9erdem@gmail.com.</p>
```

Comment:
You can make LinkedIn and the email link here.
```suggestion
      <p>📫 Reach out on <a hrtef="https://www.linkedin.com/in/bayramerdem" target="_blank">LinkedIn</a> or email me at <a href="mailto:bayram9erdem@gmail.com">bayram9erdem@gmail.com</a>.</p>
```